sample-plugin/pom.xml

```diff
@@ -15,7 +15,6 @@
         <changelist>999999-SNAPSHOT</changelist>
         <bom>weekly</bom>
         <jenkins.version>2.395</jenkins.version>
-        <kotlin.version>1.8.10</kotlin.version>
```

Comment:
#1851 was incomplete: it removed the last consumer of this property but not the property itself.